article/app/views/fragments/liveBlogNavigation.scala.html

```diff
@@ -15,7 +15,7 @@
             }
             class="liveblog-navigation__link liveblog-navigation__link--secondary liveblog-navigation__link--secondary--newer @if(pagination.newest.isEmpty) { liveblog-navigation__link--disabled }"
             data-link-name="newest page">
-                @fragments.inlineSvg("arrow-left-double", "icon", List("liveblog-navigation__link--secondary--newer__icon"))
+                @fragments.inlineSvg("arrow-left-double", "icon", List("liveblog-navigation__link--secondary--newer__icon", if(pagination.newest.isEmpty) "liveblog-navigation__link--disabled" else "liveblog-navigation__link--enabled"))
```

Comment:
Good shout @johnduffell, those names are not very BEM-y. I'll probably go with `liveblog-navigation__link__icon--disabled` and `liveblog-navigation__link__icon--enabled`